Kinvey/Kinvey/CustomEndpoint.swift

```diff
@@ -36,13 +46,77 @@ open class CustomEndpoint: NSObject {
         return request
     }
     
+    //MARK: Mappable
+    
+    /// Executes a custom endpoint by name and passing the expected parameters.
+    @discardableResult
+    internal static func execute<T: Mappable>(_ name: String, params: JsonDictionary? = nil, client: Client = sharedClient, completionHandler: CompletionTHandler<T>? = nil) -> Request {
```

Comment:
do you think we should expose headers too? request method has to be POST always and the body has to be a json object too